src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -32,6 +32,7 @@
 @SliceQACheck
 public class ChimericInstancesCheck extends AbstractQACheck {
 
+    private static final String skiplist = "resources/chimerism_reference_constraint_violations.txt";
```

Comment:
Could the name be `SKIP_LIST_FILE_PATH` since it is a class-level constant and the path to the file containing the db ids to skip rather than the actual list of db ids itself?

Reply:
or maybe it could still be a lowercase name, but moved to be a local constant in the `getDisplayName` method?

Reply:
It probably shouldn't be a constant - it might be good to be able to use different filenames for the different skiplists of different QA checks.

Reply:
I changed everything to the `SKIP_LIST_FILE_PATH` style